src/components/page/page_body/page_body.stories.tsx

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+import type { Meta, StoryObj } from '@storybook/react';
+
+import contentSvg from '../../../../src-docs/src/images/content.svg';
+import { EuiImage } from '../../image';
+import { EuiPage } from '../page';
+import { EuiPageSection } from '..//page_section';
```

Comment:
[nit] you have several imports with double slashes across various files in PR - let's remove them for consistency
```suggestion
import { EuiPageSection } from '../page_section';
```